libs/util/shared/src/lib/services/proxy.service.spec.ts

```diff
@@ -2,7 +2,7 @@ import { PROXY_PATH, ProxyService } from './proxy.service'
 import { TestBed } from '@angular/core/testing'
 
 // mock window.location
-;(global as any).window = Object.create(window)
+;(global as any).window ??= Object.create(window)
```

Comment:
Is this change intentional? I've never seen that before :open_mouth:

Reply:
It is ! But it has to be discussed. I've seen it there and in this [PR](https://github.com/geonetwork/geonetwork-ui/pull/505) that `proxy.service.spec.ts` fails with the error [TypeError: Cannot redefine property: window](https://stackoverflow.com/questions/75414699/jest-typeerror-cannot-redefine-property-window). 

I can't understand why because nothing have been edited in proxy file... If you have some hints.

Reply:
Oh ok, that's a bug in node 18, see https://github.com/geonetwork/geonetwork-ui/pull/482
which version of node do you use for running the tests?

Reply:
18.16.00 ... I'll edit those two branches and create an issue in order to find a way to upgrade node version. It seems unfortunate that we can"t upgrade it due to one test. What do you think ?

Reply:
Apparently 18.15 does not have the bug so we could use that? but it's also probably a matter of weeks before the backport happens on 18.x and then we can just switch to LTS safely :)

Reply:
Ok let's leave it like that, it actually works around the bug in a very simple way. If we add a link to the node bug report then it's all good :)